posthog/hogql_queries/web_analytics/web_overview.py

```diff
@@ -190,60 +209,68 @@ def previous_period_aggregate(
                 params=params,
             )
 
+        def add_metric_pair(
+            metrics_list: list[ast.Expr],
+            function_name: str,
+            column_name: str,
+            current_alias: str,
+            previous_alias: Optional[str] = None,
+            params: Optional[list[ast.Expr]] = None,
+        ) -> list[ast.Expr]:
+            # This could also be done using tuples like the stats_table but I will keep the protocol as close as possible: https://github.com/PostHog/posthog/blob/26588f3689aa505fbf857afcae4e8bd18cf75606/posthog/hogql_queries/web_analytics/stats_table.py#L390-L399
+            previous_alias = previous_alias or f"previous_{current_alias}"
+            metrics_list.append(current_period_aggregate(function_name, column_name, current_alias, params))
+            metrics_list.append(previous_period_aggregate(function_name, column_name, previous_alias, params))
+            return metrics_list
+
+        select: list[ast.Expr] = []
+
         if self.query.conversionGoal:
-            select = [
-                current_period_aggregate("uniq", "session_person_id", "unique_users"),
-                previous_period_aggregate("uniq", "session_person_id", "previous_unique_users"),
-                current_period_aggregate("sum", "conversion_count", "total_conversion_count"),
-                previous_period_aggregate("sum", "conversion_count", "previous_total_conversion_count"),
-                current_period_aggregate("uniq", "conversion_person_id", "unique_conversions"),
-                previous_period_aggregate("uniq", "conversion_person_id", "previous_unique_conversions"),
-                ast.Alias(
-                    alias="conversion_rate",
-                    expr=ast.Call(
-                        name="divide", args=[ast.Field(chain=["unique_conversions"]), ast.Field(chain=["unique_users"])]
-                    ),
+            # Add standard conversion goal metrics
+            add_metric_pair(select, "uniq", "session_person_id", "unique_users")
+            add_metric_pair(select, "sum", "conversion_count", "total_conversion_count")
+            add_metric_pair(select, "uniq", "conversion_person_id", "unique_conversions")
```

Comment:
Again, slightly more pythonic if you changed `add_metric_pair` to return a list with two entries and then
```suggestion
            # Add standard conversion goal metrics
            select.extend(add_metric_pair("uniq", "session_person_id", "unique_users"))
            select.extend(add_metric_pair("sum", "conversion_count", "total_conversion_count"))
            select.extend(add_metric_pair("uniq", "conversion_person_id", "unique_conversions"))
```